flocker/provision/_install.py

```diff
@@ -711,6 +772,27 @@ def task_enable_flocker_control(distribution):
         raise DistributionNotSupported(distribution=distribution)
 
 
+def task_enable_docker_plugin(distribution):
+    """
+    Enable the Flocker Docker plugin.
+
+    :param bytes distribution: The distribution name.
+    """
+    if is_centos(distribution):
+        return sequence([
+            run_from_args(['systemctl', 'enable', 'flocker-docker-plugin']),
+            run_from_args(['systemctl', START, 'flocker-docker-plugin']),
+            run_from_args(['systemctl', START, 'docker']),
+        ])
+    elif distribution == 'ubuntu-14.04':
```

Comment:
Maybe it hasn't landed yet, but another recent PR adds an `is_ubuntu` checker. Should you be using that?